__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("bad API request", () => {
+  test("returns route not found and gives a 404", () => {
+    return request(app)
+      .get("/api/categorie")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Route not found");
+      });
+  });
+});
+describe("GET /api/categories", () => {
+  test("returns an array of category objects each should have properties slug and description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(Array.isArray(categories)).toBe(true);
+        expect(categories).toHaveLength(4);
+        expect(categories[0]).toEqual({
+          description: "Abstact games that involve little luck",
```

Comment:
instead of hardcoding your test to look for this exact object in this way, make it more flexible by checking the properties for being the correct data type.
Look into how we use toMatchObject and expect.any(datatype) to form tests that will work for any object in the array, not just one you hardcode, improving the quality of your tests.